### DISCUSSION TITLE
How to install the Aspire 9 workload to use the nightly Nuget packages?

### DISCUSSION THREAD

**Discussion by wertzui** [microsoft/aspire#6208] [Q&A] [answered]
I just wanted to start using the packages from the nightly nuget.

VS instructed me to add `<Sdk Name="Aspire.AppHost.Sdk" Version="9.0.0" />` to my `....Apphost.csproj`

So I did that and now I get `The SDK 'Aspire.AppHost.Sdk/9.0.0' specified could not be found.`

What do I need to do to get it running?

**Comment by davidfowl** [comment, marked as answer]
We haven't written instructions yet but:

https://github.com/dotnet/aspire/blob/main/docs/using-latest-daily.md#install-the-nightly-net-aspire-templates

Aspire 9 supports .NET 8 as well
